src/stepFunctions/commands/visualizeStateMachine/aslVisualizationManager.ts

```diff
@@ -11,34 +11,24 @@ import { getLogger, Logger } from '../../../shared/logger'
 
 export class AslVisualizationManager extends AbstractAslVisualizationManager {
     protected readonly name: string = 'AslVisualizationManager'
-    private readonly managedVisualizations: Map<string, AslVisualization> = new Map<string, AslVisualization>()
 
     public constructor(extensionContext: vscode.ExtensionContext) {
         super(extensionContext)
     }
 
     public async visualizeStateMachine(
         globalStorage: vscode.Memento,
-        input: vscode.Uri | vscode.TextEditor | undefined
+        target: vscode.TextDocument | vscode.Uri
     ): Promise<vscode.WebviewPanel | undefined> {
         const logger: Logger = getLogger()
+        const document = target instanceof vscode.Uri ? await vscode.workspace.openTextDocument(target) : target
 
         /* TODO: Determine behaviour when command is run against bad input, or
          * non-json files. Determine if we want to limit the command to only a
          * specifc subset of file types ( .json only, custom .states extension, etc...)
          * Ensure tests are written for this use case as well.
          */
 
-        const document =
```

Comment:
is the big comment above, still needed?

Reply:
Guess not. Looks like it's talking about validation which definitely doesn't belong here.